catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/EntityResourceTest.java

```diff
@@ -466,6 +508,77 @@ protected void prepareGetWithDifferentFields(T entity) throws HttpResponseExcept
   ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
   // Common entity tests for GET operations
   ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  @Test
+  void get_entityWithAuthorizedViewMetadataOperation(TestInfo test)
+      throws HttpResponseException, JsonProcessingException {
+    if (!this.getClass().getName().endsWith("TableResourceTest")) {
```

Comment:
@harshach I got this feature (backend side) working for tables.

We can extend this to other entities if the overall approach looks good.